pkg/serviceaccount/claims.go

```diff
@@ -23,11 +23,16 @@ import (
 
 	"gopkg.in/square/go-jose.v2/jwt"
 	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/serviceaccount/metrics"
 
 	apiserverserviceaccount "k8s.io/apiserver/pkg/authentication/serviceaccount"
 	"k8s.io/kubernetes/pkg/apis/core"
 )
 
+// Expiration extension constants
+const expirationSecondsSpecial = 60*60 + 7
+const expirationExtensionHours = 24 * 365
```

Comment:
I'd expect this to be bounded to the lesser of `--service-account-max-token-expiration` or 1 year